.tekton/external-dns-operator-container-ext-dns-optr-1-3-rhel-9-pull-request.yaml

```diff
@@ -172,6 +172,25 @@ spec:
       workspaces:
       - name: basic-auth
         workspace: git-auth
+    - name: generate-labels
```

Comment:
Do we still need this task?

Reply:
I [checked it](https://github.com/openshift/external-dns-operator/pull/273#issuecomment-3327200028), we don't need `generate-labels` task, `clone-repository` has what we need for the git repository and the commit.